### PR TITLE
#1: fixed route position

### PR DESCRIPTION
**Pull request by umbarutiba**
@alexkontsevoy done, please check

### CODE REVIEW

File: core/utility/ast-utils.ts [hunk 1]
```diff
@@ -40,18 +40,23 @@ function _buildRoute(options: BuildRouteOptions): string {
 
   const loadChildren = `() => import('${relativeRouteModulePath}').then((module) => module.${routeModule}Module)`;
 
-  let route = `{
-    path: '${routePath}',
-    loadChildren: ${loadChildren}
-  }`;
+  let route = (options.isChildren)
+    ? (options.isFirstRoute)
```
Comment (astorozhevsky) [inline core/utility/ast-utils.ts]: @umbarutiba please refactor this ternary operator. It looks very dirty

File: core/utility/ast-utils.ts [hunk 2]
```diff
@@ -40,18 +40,23 @@ function _buildRoute(options: BuildRouteOptions): string {
...
 
-  if (options.isFirstRoute) {
+  if (options.isFirstRoute && !options.isChildren) {
     route = `\n  ${route}\n`;
   }
 
   return route;
 }
 
+function _removeStartComma(_str: string): string {
```
Comment (astorozhevsky) [inline core/utility/ast-utils.ts]: @umbarutiba `_str` -> `str`

File: core/utility/ast-utils.ts [hunk 3]
```diff
@@ -40,18 +40,23 @@ function _buildRoute(options: BuildRouteOptions): string {
...
+  return _str.slice(1);
```
Comment (astorozhevsky) [inline core/utility/ast-utils.ts]: @umbarutiba if you pass string `test string` without comma at the start, this method will remove the `t` char. Need to check if the first char is comma.

File: core/utility/ast-utils.ts [hunk 4]
```diff
@@ -88,6 +93,26 @@ function _addSymbolToNgModuleMetadata(options: AddSymbolToNgModuleMetadataOption
   };
 }
 
+function _getChildrenRoutesPosition(source: ts.SourceFile): number | null {
+  const routes = findNodes(source, ts.SyntaxKind.VariableDeclarationList).find((item) => item.getText().includes('const routes: Routes'));
+  if (routes === undefined) {
+    return null;
+  }
+  const childrenRoutes = findNodes(routes, ts.SyntaxKind.PropertyAssignment).find((item) => item.getText().includes('children:'));
+  if (childrenRoutes === undefined) {
```
Comment (astorozhevsky) [inline core/utility/ast-utils.ts:104]: @umbarutiba can we create the property `children` if it doesn't exist?

**Review by astorozhevsky — CHANGES_REQUESTED** [review]

File: core/utility/ast-utils.ts (hunk 4 above)
Comment (umbarutiba) [reply core/utility/ast-utils.ts:104]: as discussed

**Comment by umbarutiba** [comment]
@astorozhevsky fixed, please check

**Comment by dmitryusenko** [comment]
@astorozhevsky @umbarutiba
Looks good to me.

**Review by astorozhevsky — APPROVED** [review]

**Pull request merged**